src/messages/MOSDPing.h

```diff
@@ -80,6 +99,8 @@ class MOSDPing : public Message {
     ::encode(op, payload);
     ::encode(peer_stat, payload);
     ::encode(stamp, payload);
+    bufferlist size_bl(MAX(min_message_size - payload.length(), 0));
```

Comment:
pretty sure this doesn't actually put anything in the list; the ctor arg just sizes append_buffer (how big we expect the bl to get)